macros/REST_Geant4_ViewGeometry.C

```diff
@@ -15,12 +18,12 @@ using namespace std;
 //*** This macro might need update/revision.
 //***
 //*******************************************************************************************************
-Int_t REST_Geant4_ViewGeometry(TString fName, TString option = "") {
-    cout << "Filename : " << fName << endl;
+Int_t REST_Geant4_ViewGeometry(const char* filename, const char* option = "") {
+    cout << "REST View Geometry Macro called with input: " << filename << endl;
 
     TGeoManager* geo = nullptr;
-    if (TRestTools::isRootFile((string)fName)) {
-        TFile* fFile = new TFile(fName);
+    if (TRestTools::isRootFile(filename)) {
+        TFile* fFile = new TFile(filename);
```

Comment:
There is any reason to use a pointer here? Plesase consider to move to `TFile fFile(filename)`

Reply:
I updated the `TFile` initializer to `TFile::Open` which I think is the recommended way, still it returns a pointer which I think is the recommended way to handle files (problems if you do not work in the heap I guess).

Reply:
Indeed TFile::Open is recomended if you are using a pointer, but I am afraid that the underlying ROOT code is using new as well. Alternatively you can do `std::unique_ptr<TFile> fFile( TFile::Open(filename) );`

However, in this particular case I think is fine to work in the stack since you just get a pointer from the TFile. I think issues arises when you want to handle the TFile e.g. save, update, ...